### PR TITLE
Feat/githubfirststeps

### PR DESCRIPTION
**Pull request by gilgda**
Added the github firststeps tutorial without the disputed squash tutorial

### CODE REVIEW

**Comment by Lordnoname** [comment]
Please add the tutorial to the guide.json

File: src/content/guides/GITHUB/CLONE/CLONE.MD.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+1. Navigiere in das Verzeichnis in dem du das Github-Projekt abspeichern willst
+
+2. Kopiere nun die URL der Projektseite neben dem HTTPS-Text 
+
+3. �ffne jetzt das Terminal in dem gew�nschten Verzeichnis
```
Comment (Lordnoname) [inline src/content/guides/GITHUB/CLONE/CLONE.MD.txt]: What happened with your umlauts?

File: src/content/guides/GITHUB/MAKE-CHANGES/MAKE-CHANGES.MD.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+1. �ffne das Terminal in dem Projektordner
```
Comment (Lordnoname) [inline src/content/guides/GITHUB/MAKE-CHANGES/MAKE-CHANGES.MD.txt]: Same here

File: src/content/guides/GITHUB/MAKE-CHANGES/MAKE-CHANGES.MD.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+2. Gib `git pull` ein um alle Ver�nderungen des Projektes herunterzuladen
```
Comment (Lordnoname) [inline src/content/guides/GITHUB/MAKE-CHANGES/MAKE-CHANGES.MD.txt]: Same as above

File: src/content/guides/GITHUB/MAKE-CHANGES/MAKE-CHANGES.MD.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+3. Jetzt sind die Ordner ver�nderbar
```
Comment (Lordnoname) [inline src/content/guides/GITHUB/MAKE-CHANGES/MAKE-CHANGES.MD.txt]: Same as above

File: src/content/guides/GITHUB/MAKE-CHANGES/MAKE-CHANGES.MD.txt [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+4. Wenn alle Ver�nderungen gemacht und abgespeichert wurden, gib `git add .` in das Terminal ein um deine Ver�nderungen zum Source Control Management hinzuzuf�gen
```
Comment (Lordnoname) [inline src/content/guides/GITHUB/MAKE-CHANGES/MAKE-CHANGES.MD.txt]: Same as above

File: src/content/guides/GITHUB/MAKE-CHANGES/MAKE-CHANGES.MD.txt [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+
+5. Nun m�ssen die �nderungen mit dem Befehl `git commit -m [Nachricht]` commited (�bergeben) werden, benutze hierf�r eine passende Nachricht wie diese: http://goo.gl/U2hMgE
```
Comment (Lordnoname) [inline src/content/guides/GITHUB/MAKE-CHANGES/MAKE-CHANGES.MD.txt]: Same as above

**Comment by gilgda** [comment]
Problem unknown, any ideas how to solve it? (It seems ok when I want to change it and also in my files)

**Comment by DevVersion** [comment]
@gilgda Tried saving the file in UTF-8? Maybe my ISO2UTF bash script can solve the problem ([here](https://gist.github.com/DevVersion/4c69407647d00454989b)). Please check the encoding of the file and write it here.

**Comment by DevVersion** [comment]
@gilgda Nice :+1: You fixed the encoding. Now one thing, can you please rename the `.txt` files to end with `.MD`

**Pull request merged**